tsdb/config.go

```diff
@@ -98,8 +100,14 @@ type Config struct {
 }
 
 func NewConfig() Config {
+	defaultEngine := DefaultEngine
+	if engine := os.Getenv("INFLUXDB_DATA_ENGINE"); engine != "" {
```

Comment:
You set this here because the tests don't actually go through `cmd/influxd/run/command.go` I assume so the env var actually isn't applied automatically in tests?